### PR TITLE
Only run specified events when `$args` are passed with `--due-now`

### LINKED ISSUE #10: Including --due-now with a specific cron event should only run that cron
From https://github.com/wp-cli/wp-cli/issues/4020

> `:~# wp-cli cron event run --quiet publish_future_post --due-now` will go through all cron jobs that are due now and run them. Expected result: limited to those cron jobs that are actually specified, it's written as an option and not an argument after all.

### PR DESCRIPTION
**Pull request by danielbachhuber**
Fixes #10

### CODE REVIEW

File: src/Cron_Event_Command.php [hunk 1]
```diff
@@ -225,22 +225,26 @@ public function run( $args, $assoc_args ) {
 			WP_CLI::error( $events );
 		}
 
+		$hooks = wp_list_pluck( $events, 'hook' );
+		foreach( $args as $hook ) {
+			if ( ! in_array( $hook, $hooks ) ) {
```
Comment (schlessera) [inline src/Cron_Event_Command.php]: It is recommended to always use strict mode with `in_array()` (third argument set to `true`).

File: src/Cron_Event_Command.php [hunk 2]
```diff
@@ -225,22 +225,26 @@ public function run( $args, $assoc_args ) {
...
+				WP_CLI::error( sprintf( "Invalid cron event '%s'", $hook ) );
+			}
+		}
+
 		if ( \WP_CLI\Utils\get_flag_value( $assoc_args, 'due-now' ) ) {
 			$due_events = array();
 			foreach( $events as $event ) {
+				if ( ! empty( $args ) && ! in_array( $event->hook, $args ) ) {
```
Comment (schlessera) [inline src/Cron_Event_Command.php]: Strict mode recommended as well.

Comment (danielbachhuber) [inline src/Cron_Event_Command.php]: a766d711

File: src/Cron_Event_Command.php (hunk 1 above)
Comment (danielbachhuber) [inline src/Cron_Event_Command.php]: a766d711

**Pull request merged**